apps/iotune/iotune.cc

```diff
@@ -745,6 +745,7 @@ int main(int ac, char** av) {
         ("random-read-io-buffer-size", bpo::value<unsigned>()->default_value(0), "force buffer size for random read")
         ("force-io-depth", bpo::value<unsigned>()->default_value(0), "force io depth to a certain size (overriding auto detection logic)")
         ("get-best-iops-with-buffer-sizes", bpo::value<std::vector<uint64_t>>()->default_value(std::vector<uint64_t>{}, ""), "run IOPS tests with buffer sizes passed as arg and choose the best combination")
+        ("sequential_buffer_size", bpo::value<uint64_t>()->default_value(1 << 20), "Set the sequential buffer size used for read/write bandwidth calculation. Default is 1MiB")
```

Comment:
Random IO buffer sizes have separate options, so why should sequential not have separate too?